templates/keepalived.conf.j2

```diff
@@ -164,6 +164,9 @@ vrrp_instance {{ name }} {
   {% if instance.notify_stop is defined %}
   notify_stop "{{ instance.notify_stop }}"
   {% endif %}
+  {% if instance.smtp_alert is defined and instance.smtp_alert | bool %}
+  smtp_alert # Send email notification during state transition
```

Comment:
Do we need that comment templated? Can't we just have it as a jinja comment instead? Is that comment necessary? It sounds obvious to me what smtp_alert does.

Reply:
We don't need it, if you don't want it. I was having a looking at ```{% if instance.use_vmac is defined and instance.use_vmac | bool %}``` in the same file and there i found a comment.

Reply:
Yeah that's more of a nit than anything. Don't you worry about that :)